test2.rb

```diff
@@ -0,0 +1,26 @@
+a = "bbb"
+
+def foo
+    'a'
+end
+
+aaa;
+
+not 1
+
+ l = lambda {|a| a }
+ l.(x)
+l[x]
+l.call(x)
+
+  def b(x)
+    
+  end 
+
+
+  ['x','b',"c"]
+    [x,a,b,c,d,e]
+    
+    foo (  x)
+
```

Comment:
__[development][rubocop]__
* Prefer the use of `lambda.call(...)` over `lambda.(...)`. [:ref](https://github.com/bbatsov/ruby-style-guide#proc-call)
* Line is too long. [131/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)

<!-- Comment by [SideCI](https://sideci.com) -->